ui/src/Logout.tsx

```diff
@@ -28,9 +35,19 @@ const Logout: FC<WithRouterProps> = ({router}) => {
   }
 
   useEffect(() => {
+    reset()
     handleSignOut()
   }, [])
   return null
 }
 
-export default ErrorHandling(withRouter<WithRouterProps>(Logout))
+const mdtp = {
+  reset: reset,
+}
+
+export default ErrorHandling(
```

Comment:
The main reason for requesting the name change here is twofold:

1. to be more explicit on what we're resetting without having to investigate
2. To reduce confusion about whether `reset` is being called after being imported or as a prop.